src/sas/qtgui/Calculators/GenericScatteringCalculator.py

```diff
@@ -704,13 +723,20 @@ def loadFile(self):
             load_nuc = self.sender() == self.cmdNucLoad
             # request a file from the user
             if load_nuc:
-                f_type = """
-                    All supported files (*.SLD *.sld *.pdb *.PDB, *.vtk, *.VTK);;
-                        SLD files (*.SLD *.sld);;
-                        PDB files (*.pdb *.PDB);;
-                        VTK files (*.vtk *.VTK);;
-                        All files (*.*)
-                """
+                if self.model.type is ComputationType.SAXS:
+                    f_type = """
```

Comment:
To cover all cases without a nested if and no repetition (this needs a little modification):
```suggestion
cif = " *.CIF *.cif"
cif_files = "CIF files (*.CIF *.cif);;\n        "

use_cif = self.model.type is ComputationType.SAXS

f_type = f"""
    All supported files (*.SLD *.sld *.pdb *.PDB, *.vtk, *.VTK{cif if use_cif else ""});;
        SLD files (*.SLD *.sld);;
        PDB files (*.pdb *.PDB);;
        VTK files (*.vtk *.VTK);;
        {cif_files if use_cif else ""}All files (*.*)
"""
```

Reply:
Good suggestion. I'm not sure if we should change it though, as the SAXS fitter is only supposed to live in the GSC window temporarily (however long that may be). I have therefore tried to make the overall changes to the file as minimal as possible (like my nested `if` which can easily be removed again).